### PR TITLE
Get help for partial command

### LINKED ISSUE #94: Improve output of help commands
This issue will be a little underdefined, but I'd like to start a discussion around this. Is there a way to improve the output of the help commands? Can discoverability be improved? We are using the default help from `oclif`, and just adding the relevant data. Can we make a custom help function more suitable for this project?

### PR DESCRIPTION
**Pull request by uivlis**
Fixes #94. Shows help menu for any command that is passed without parameters.

### CODE REVIEW

**Comment by ChrisChinchilla** [comment]
@fvictorio Is this basically what you were looking for? There's not much text-wise for me to review here, so more in your court

File: src/commands/contract/deploy.ts [hunk 1]
```diff
@@ -29,26 +30,31 @@ export class DeployCommand extends NetworkCommand {
...
-
-    try {
-      networkUrl = this.getNetworkUrl(flags)
-
-      const { bin } = args
-      const { pk, 'confirmation-blocks': confirmationBlocks } = flags
-      if (!pk) {
-        this.error('You have to specify a private key using --pk', { exit: 1 })
-        return
+    console.log(args, flags)
```
Comment (fvictorio) [inline src/commands/contract/deploy.ts]: This shouldn't be here, should it?

**Comment by fvictorio** [comment]
I have two concerns here.

1. The commands should return with 1 to indicate that they failed. This can be done just adding an exit line after each `this._help()`. Alternatively, we could have a base command with a method like `showHelpAndExit()` (or something less verbose) that does this.

2. This won't show the help when the command is invalid but not empty, right? For example `eth method:send erc20` needs two more args. This seems somewhat harder to do the right way, so I'm OK with not doing it for now.

**Comment by ChrisChinchilla** [comment]
Good to go!

**Comment by fvictorio** [comment]
@uivlis can you squash everything into a single commit with a message like `chore: improve output of help commands`?

**Comment by uivlis** [comment]
@fvictorio I'm not sure what I did, hope it's squashed as requested.

**Comment by fvictorio** [comment]
No, I don't think that worked, but don't worry, I can do it later for you. But first, could you please fix the linting errors?

**Comment by uivlis** [comment]
Again, I'm not sure how to fix the linting errors with this sole exception, that I committed. For example, the first error is:
```
ERROR: /home/travis/build/protofire/eth-cli/src/commands/abi/methods.ts:2:1 - Imports from this module are not allowed in this group.  The expected groups (in order) are: libraries, parent directories, current directory.
```
But the imports are already in order:
```
import { Command } from '@oclif/command'
import { isEmptyCommand } from '../../helpers/checkCommandInputs'
```
I couldn't solve the errors in these cases.

**Comment by fvictorio** [comment]
Thanks @uivlis!

**Comment by uivlis** [comment]
@fvictorio No problem, happy to help! Also, waiting for the funds on Gitcoin to be released 😸

**Comment by ChrisChinchilla** [comment]
@uivlis Bounty will be paid out later today (person is in US time zone)

**Pull request merged**